gui/src/ocpn_frame.cpp

```diff
@@ -5498,8 +5498,7 @@ void MyFrame::OnFrameTimer1(wxTimerEvent &event) {
     GPSData.nSats = g_SatsInView;
 
     wxDateTime tCheck((time_t)m_fixtime);
-
-    if (tCheck.IsValid())
+    if (m_fixtime != -1 && tCheck.IsValid())
```

Comment:
`m_fixtime == -1` when no data is available.
`-1` is considered a valid date for `wxDataTime`.